### PR TITLE
500 content edit

### PR DESCRIPTION
**Pull request by jameshupp**
## Changes
* More 500-specific intro

## Review
* @sarahsimpson09 
* @sebworks 

## Notes
* I don't think I've mis-described a 500 error.
* Until further notice, the FEWD I'm going to include as reviewer is the most recent person to edit it.

### CODE REVIEW

File: _layouts/500.html [hunk 1]
```diff
@@ -16,10 +16,9 @@
     <div class="error_copy_container">
         <header class="error_header u-mt30">
             <h1 class="superheader">500 internal server error</h1>
-            <p class="short_desc">We can’t find the page you were
-                looking for, but here are
-                a number of other ways you
-                might try to get there.
+            <p class="short_desc">That didn’t work the way it was supposed to. You
+                might want to try again in a little bit. We'll try to resolve
```
Comment (anselmbradford) [inline _layouts/500.html]: `We'll` > `We’ll`

File: _layouts/500.html [hunk 2]
```diff
@@ -35,10 +34,10 @@ <h1 class="superheader">500 internal server error</h1>
                         If the problem persists please let us know.
                     </p>
                     <a class="jump-link jump-link__right" href="/contact-us">
-                        Visit contact us page
+                        Contact us
```
Comment (jimmynotjim) [inline _layouts/500.html]: Same question here as on the 404 page.

**Comment by jimmynotjim** [comment]
@Scotchester do your comments on the 404 changes apply to this one as well?

**Comment by Scotchester** [comment]
Yes, but the error code has not been removed from this page.

**Comment by jameshupp** [comment]
By definition, isn't it impossible to be more specific in the title and description for a 500 error?

**Comment by Scotchester** [comment]
I was thinking more about "500" being in the content of the page than the title/description thing. Those are probably fine as they are.

**Comment by Scotchester** [comment]
So I guess we're good here, too.

**Pull request merged**